cmd/query/app/static_handler.go

```diff
@@ -76,15 +76,15 @@ type loadedConfig struct {
 
 // NewStaticAssetsHandler returns a StaticAssetsHandler
 func NewStaticAssetsHandler(staticAssetsRoot string, options StaticAssetsHandlerOptions) (*StaticAssetsHandler, error) {
-	assetsFS := ui.StaticFiles
-	if staticAssetsRoot != "" {
-		assetsFS = http.Dir(staticAssetsRoot)
-	}
-
 	if options.Logger == nil {
 		options.Logger = zap.NewNop()
 	}
```

Comment:
Would any test fail if we remove this if statement? If not, we should remove it.

Reply:
removed